pkg/http/api.go

```diff
@@ -1341,7 +1359,6 @@ func (a *api) onDirectActorMessage(reqCtx *fasthttp.RequestCtx) {
 		log.Debug(msg)
 		return
 	}
-	defer resp.Close()
```

Comment:
Why is this line removed?

Reply:
It was moved up, to make sure it is closed even in case of error.